notifications/client.go

```diff
@@ -24,7 +26,13 @@ func CreateClient() Client {
 	client.Transport = &http.Transport{
 		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
 	}
-	return Client{httpClient: client}
+
+	id := os.Getenv("KUBE_MONKEY_ID")
+
+	return Client{
```

Comment:
since this is an env var, can we just read it next to the code that's using it with the `id := os.Getenv("KUBE_MONKEY_ID")` instead of adding it to the client structure?

Reply:
Yes of course